vic/vic_run/src/surface_fluxes.c

```diff
@@ -225,11 +224,9 @@ surface_fluxes(bool                 overstory,
     double            store_Raut;
     double            store_NPP;
 
-    if (options.CLOSE_ENERGY) {
-        MAX_ITER_GRND_CANOPY = 10;
-    }
-    else {
-        MAX_ITER_GRND_CANOPY = 0;
+    if (!options.CLOSE_ENERGY) {
+        // if not closing energy balance, don't iterate
+        param.MAX_ITER_GRND_CANOPY = 0;
```

Comment:
can we move this change to the parameters initialization? No need to reset this variable each timestep.

Reply:
I considered doing that but moving it to `vic/drivers/shared_all/src/initialize_parameters.c` requires using the `options` struct in that routine, which we don't currently do - do you think that's fine?

Reply:
yes, I think that it probably cleanest.

Reply:
I've updated this as you suggested - I'll ping you once the tests pass and it's ready for another review.